### DISCUSSION TITLE
correlateTrajectories

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#804] [Questions / Documentation]
I would like to see if it is possible to find all the genes with expression correlated with the motif accessibility, I was wondering if ArchR has any implementation for that?

**Comment by rcorces** [comment]
There is a vignette on this in the manuscript. Ext. Data Fig 8f:
![image](https://user-images.githubusercontent.com/10343046/120955239-c93e4f80-c705-11eb-99df-0515936a18bc.png)
This uses gene score but same underlying principle.
There us no function to do this for you though so the implementation would be up to you but you can find the code we used here:
https://github.com/GreenleafLab/ArchR_2020/blob/6744d22be46a11ed6732e1eb9aaf9afb429a3e87/Code/ArchR_PostAnalysis_Large_Heme_Analysis.R#L157
